gen3/lib/aws.sh

```diff
@@ -322,6 +324,16 @@ EOM
     return 0
   fi
 
+  # else ...
+  if [[ "$GEN3_WORKSPACE" =~ _es$ ]]; then
+      commonsName=${GEN3_WORKSPACE//_es/}
+      cat - <<EOM
```

Comment:
Would you verify the `//_es/` trick works in `zsh`?  Zsh drives me crazy ...

Reply:
my environment in cdistest is zsh and it works just fine. Also tested it locally before using it.

So, yeah, it works just fine,.